galleria/themes/classic/galleria.classic.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
Since `galleria` also contains minified file, please remove this one, as it's not required.